### ISSUE TITLE
Questions about training and stuff

### ISSUE THREAD

**Issue by mirth** [jxmorris12/vec2text#100]
1. I'm training `vec2text` with `t5-base` with a some embedder, the questions are: what are recommended number of a training epochs & expected loss of a trained model?
2. Is it ok to share trained `vec2text` model on the huggingface?

**Comment by jxmorris12** [comment]
I think these hyperparameters are all in [the paper](https://arxiv.org/abs/2310.06816); it depends on how much data you have in particular and what size embedding. As a general rule, you will likely want at least 10 epochs.

Yes, you can share the models -- [we did this](https://huggingface.co/vec2text) as well.
